sonic-slave-stretch/Dockerfile.j2

```diff
@@ -329,7 +329,7 @@ RUN export VERSION=1.14.2 \
  && rm go$VERSION.linux-*.tar.gz
 
 RUN pip3 install --upgrade pip
-RUN pip2 install --upgrade pip
+RUN pip2 install --upgrade 'pip<21'
```

Comment:
not sure why need this

Reply:
This is simply to align the way we install/upgraded pip everywhere